### PR TITLE
Move perf functions in a shared utils file

### PR DESCRIPTION
**Pull request by Copons**
## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/e2e-test-utils/src/index.js [hunk 1]
```diff
@@ -52,6 +52,14 @@ export {
 } from './observe-focus-loss';
 export { openDocumentSettingsSidebar } from './open-document-settings-sidebar';
 export { openPublishPanel } from './open-publish-panel';
+export {
```
Comment (gziolo) [inline packages/e2e-test-utils/src/index.js]: What's the motivation to make those utils part of the public API? Can they be located in the subfolder of `packages/e2e-tests`?

Comment (Copons) [reply packages/e2e-test-utils/src/index.js]: 🤔 No particular reasons, to be honest.
I'm not familiar with Guten's e2e, and just figured that utils are supposed to go in `e2e-test-utils`. 😅 

As far as I can see, these utils are only relevant for the performance tests, so my first thought was to create a utils folder into `packages/e2e-tests/specs/performance`, but then I couldn't find any other "local utils".

Comment (gziolo) [reply packages/e2e-test-utils/src/index.js]: We can start with local `utils` in the top level next to `specs` 😄 

You can also try inside `specs/performance` unless Jest picks them up as tests.

**Comment by Copons** [comment]
I'm not able to work on it ATM, so I'm just gonna close this PR for now. Feel free to reopen and proceed if there's a need, though! 🙂

**Pull request closed without merging**